core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -247,7 +247,7 @@ ByteBuffer readKey(int entryIndex) {
      * The thread-local ByteBuffer can be reused by different threads, however as long as
      * a thread is invoked the ByteBuffer is related solely to this thread.
      */
-    void setKeyRefer(int entryIndex, OakRReference keyReferBuffer) {
+    void setKeyRefer(int entryIndex, OakRBufferStream keyReferBuffer) {
```

Comment:
I am sorry I am against the "OakRBufferStream" name... Why stream? It is used also in EntryIterator and in KeyIterator (not only Key/EntryStreamIterator). This is just the buffer for read access for keys, so any combination of Oak-R-Key-Buffer (with or without Impl) should work

Reply:
fixed